### ISSUE TITLE
Cannot install ncl at all

### ISSUE THREAD

**Issue by maddenp** [conda-forge/ncl-feedstock#26]
```
% conda create -n foo -c conda-forge ncl
Fetching package metadata .............
Solving package specifications: .

UnsatisfiableError: The following specifications were found to be in conflict:
  - ncl
Use "conda info <package>" to see the dependencies for each package.
```
If I `s/ncl/gdal/`, for example, it's fine, so I can't immediately see a way to blame myself.

`conda search -c conda-forge ncl` turns up the expected options. Asking for e.g. `ncl=6.4.0` doesn't help, either.

**Comment by khallock** [comment]
I've narrowed the issue down to 3 dependencies:

```
$ conda create -n foo -c conda-forge hdfeos5 libgdal=2.1 proj4=4.9.3                   
Fetching package metadata .............
Solving package specifications: .

UnsatisfiableError: The following specifications were found to be in conflict:
  - hdfeos5
  - libgdal 2.1*
  - proj4 4.9.3*
Use "conda info <package>" to see the dependencies for each package.
```

@ocefpaf, do you have any ideas?

**Comment by ocefpaf** [comment]
`hdfeos5` fails with we tried to build it with `hdf5 1.10.11`. We could get versions of all the dependencies with `hdf5 1.8.18` to re-build `ncl`, but that is quite time consuming.

Do you know if we need a patch or something to get `hdfeos5` to work with latest `hdf5`?
If not are you OK  dropping `hdfeos5` support in the next release?

**Comment by maddenp** [comment]
I'd be happy to use a workaround to install `ncl` (maybe with extra pinning?) in the meantime if anybody can suggest a recipe. Thanks in advance if that's possible.

**Comment by maddenp** [comment]
I tried doing a manual build from the feedstock on my system (`conda build -c conda-forge recipe`), removing `hdfeos5` from `meta.yaml` and unpinning the `hdf` package, but the tests at the end failed:
```
++ ncl -V
/lscratch/pmadden/miniconda/conda-bld/ncl_1513346579262/test_tmp/run_test.sh: line 7: ncl: command not found
+ test '' = 6.4.0
Tests failed for ncl-6.4.0-blas_openblas_6.tar.bz2 - moving package to /lscratch/pmadden/miniconda/conda-bld/broken
WARNING:conda_build.build:Tests failed for ncl-6.4.0-blas_openblas_6.tar.bz2 - moving package to /lscratch/pmadden/miniconda/conda-bld/broken
removing: ncl-6.4.0-blas_openblas_6.tar.bz2
TESTS FAILED: ncl-6.4.0-blas_openblas_6.tar.bz2
```
Any idea as to why `ncl` wouldn't be available at this point in the build?

**Comment by lukeconibear** [comment]
From https://www.ncl.ucar.edu/Download/conda.shtml:

NOTE: There is a known issue with the NCL conda-forge package that causes an "UnsatisfiableError" in conda:

  UnsatisfiableError: The following specifications were found to be in conflict:
    - ncl
As of 2017-12-20, we have a workaround for this issue that forces a "broken" version of GDAL to be installed:
  conda create -n ncl_fix -c conda-forge -c conda-forge/label/broken ncl poppler=0.52 xerces-c=3.1 gsl

**Comment by ocefpaf** [comment]
Should be fixed by #27 

Please re-open if you are still experiencing issues.
